frontend/public/components/_masthead.scss

```diff
@@ -77,6 +77,11 @@
       display: flex;
     }
   }
+
+  &__updates {
```

Comment:
Oops. This actually doesn't work. The pf rules are still overriding because of these rules (truncated):
```sass
.navbar-pf-vertical {
  .nav {
     .nav-item-iconic {
       color: $navbar-pf-vertical-color;
      .caret,
      .fa,
      .pficon {
        color: $navbar-pf-vertical-color;
      }
    }
  }
}
```

Reply:
Does this work?

```css
&__updates .fa {
```

Reply:
Yeah, this works. I'm gonna also add a comment to maybe revisit this later to come up with a better solution for different color icons in the masthead. It could become a problem again if we ever want to add any other notification type icons in the masthead.

Reply:
It probably changes completely with pf4 as well (@rhamilto fyi)

Reply:
> It probably changes completely with pf4 as well (@rhamilto fyi)

Yep.  But no conflicts as `_masthead.scss` is deleted in the PF4 branch, and the PF4 styles are mostly correct.

Reply:
Do you color the icon in pf4?

Reply:
> Do you color the icon in pf4?

No, not yet as it was white like master when I put it in a couple days ago.